sharepoint/templates/istio/gateway.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if and ( .Values.ingress.istio.enabled ) ( not .Values.ingress.istio.existingGateway ) }}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
```

Comment:
I am not sure this one service requires its own Gateway. Couldn't you have one gateway for all of scp?

Reply:
Yes, trying to support two modes:  1) As part of the scp cluster 2) external cluster to scp.

You can share the same gateway as scp by setting the .Values.ingress.istio.existingGateway value